src/model/Model.cpp

```diff
@@ -1518,7 +1518,13 @@ namespace model {
     buildingStory.setNominalFloortoFloorHeight(3);
 
     // create spaces from floor print
-
+    // GG 10/26/21 - Change the spac egeneration to use fromFloorPrint instead of using Clone
```

Comment:
Clone would give each object a new unique UUID handle but that should not matter if you are properly sorting the surfaces by name.  Another good test would be to write the model to file and re-read it back in, that is a typical use case.

Reply:
I think what I was trying to do here was create what I know to be identical models in OpenStudio and export them to gbxml and then using an external file comparison verified that the files are identical. When I used Clone the order of the surfaces came back different each time so the files would never be the same (i.e. for example Surface 5 in one model was a floor and in the other model a ceiling) so I used fromfloorPrint to generate the spaces instead.